app/brave_main_delegate.cc

```diff
@@ -139,6 +143,12 @@ void BraveMainDelegate::PreSandboxStartup() {
       base::FilePath(FILE_PATH_LITERAL("/etc/brave/policies")), true, false);
 #endif
 
+#if BUILDFLAG(IS_LINUX)
+  // Ensure we have read the Linux distro before the process is sandboxed.
```

Comment:
the comment here is good, but it would be better to encapsulate this in some adblock utility or static method so it doesn't get orphaned if we make some future change that no longer requires it